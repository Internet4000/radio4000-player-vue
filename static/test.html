<!doctype html>
<html>
	<head>
		<meta charset="utf-8">
		<meta name="viewport" content="width=device-width, initial-scale=1">
		<title>Test Radio4000 player</title>
		<style>
			/* styles for this demo */
			body {
				margin: 0;
				padding: 2rem;
			}
		</style>
	</head>
	<body>

		<radio4000-player channel-slug="good-time-radio"></radio4000-player>

		<!-- <script src="dist/radio4000-player.js"></script> -->
		<script src="dist/radio4000-player.min.js"></script>

	</body>
</html>
