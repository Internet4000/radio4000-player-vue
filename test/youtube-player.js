import test from 'ava'
import Vue from 'vue'
import Component from '../src/YoutubePlayer.vue'

test('renders', t => {
	const vm = new Vue(Component).$mount()
	const tree = {
		$el: vm.$el.outerHTML
	}
	t.snapshot(tree)
	const $ = selector => vm.$el.querySelectorAll(selector)
	t.is($('.ytplayer').length, 1, 'it has an empty element for the yt player to replace')
})
