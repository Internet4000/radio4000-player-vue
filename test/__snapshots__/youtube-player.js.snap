// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
Object {
  "$el": "<div id=\\"YoutubePlayer\\"><div class=\\"Ratio\\"></div><div class=\\"ytplayer\\"></div></div>",
}
`;
